.travis.yml

```diff
@@ -108,7 +108,9 @@ script:
 - if [ $RUN_E2E_TESTS_LIBRARY_PROD == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="library" --prod_env; fi
 - if [ $RUN_E2E_TESTS_MISC_PROD == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="misc" --prod_env; fi
 - if [ $RUN_E2E_TESTS_USERS_PROD == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="users" --prod_env; fi
-- if [ $RUN_E2E_TESTS_EMBEDDING_PROD == 'true' ]; then travis_retry bash scripts/run_e2e_tests.sh --suite="embedding" --prod_env; fi
+# This is commented out because in production exploration can not be reloaded.
```

Comment:
Hi @MAKOSCAFEE -- I know we talked about commenting this out before, but would it be possible to fix this issue as well in this PR? The reason for the change in direction is because we've recently encountered at least one issue in the embedding tests that was caught by the e2e test, and so I'm a bit hesitant to drop the coverage even temporarily.

Would that be OK?